Telemetry/Telemetry.cpp

```diff
@@ -216,6 +236,45 @@ namespace WPEFramework
             returnResponse(true);
         }
 
+        uint32_t Telemetry::uploadLogs(const JsonObject& parameters, JsonObject& response)
+        {
+            LOGINFOMETHOD();
+#ifdef HAS_RBUS
+            if (RBUS_ERROR_SUCCESS != rbusHandleStatus)
+            {
+                rbusHandleStatus = rbus_open(&rbusHandle, RBUS_COMPONENT_NAME);
+            }
+
+            if (RBUS_ERROR_SUCCESS == rbusHandleStatus)
+            {
+                int ret = rbus_setBoolean(rbusHandle, T2_ON_DEMAND_REPORT, true);
+                if (RBUS_ERROR_SUCCESS != ret)
+                {
+                    std::stringstream str;
+                    str << "Failed to set " << T2_ON_DEMAND_REPORT << ": " << ret;
+
+                    LOGERR("%s", str.str().c_str());
+                    response["message"] = str.str();
+                    returnResponse(false);
+                }
+            }
+            else
+            {
+                std::stringstream str;
+                str << "rbus_open failed with error code " << rbusHandleStatus;
+
+                LOGERR("%s", str.str().c_str());
+                response["message"] = str.str();
+                returnResponse(false);
+            }
+#else
+            LOGERR("No RBus support");
+            response["message"] = "No RBus support";
+            returnResponse(false);
+#endif
+            returnResponse(true);
```

Comment:
new apis should follow JSON RPC spec https://www.jsonrpc.org/specification#response_object

fyi in case of error no response object is sent

please return error code and document it in the schema